features/git-town/configuration/configuration.feature

```diff
@@ -9,28 +9,54 @@ Feature: listing the configuration
     Given I have configured the main branch name as "main"
     And my non-feature branches are "qa" and "staging"
     When I run `git town config`
-    Then I see "Main branch: main"
-    And I see "Non-feature branches: qa, staging"
+    Then I see
+      """
+      Main branch:
+      main
```

Comment:
I really dislike having multiple branch names on a single line. Its very hard to read in my mind and I want to follow git-branch's example and only have one per line.

You are actually allowed to have commas in your branch names (though I doubt 99% of the time anyone does). Because of this I was planning on updating the storage of non feature branches to use whitespace separator. Will do make an issue for that